VKSample/VKSample/Modules/FriendPhotosGalleryModule/Cells/FriendPhotoGalleryCollectionViewCell.swift

```diff
@@ -5,27 +5,16 @@ import UIKit
 
 /// Cell with friend's photo.
 final class FriendPhotoGalleryCollectionViewCell: UICollectionViewCell {
-    // MARK: - Private Constants.
-
-    private enum Constants {
-        static let imagePlaceholderString = "imagePlaceholder"
-    }
-
     // MARK: - Private @IBOutlet.
 
     @IBOutlet private var friendPhotoImageView: UIImageView!
     @IBOutlet private var backgroundPhotoView: UIView!
 
     // MARK: - Public methods.
 
-    func configure(imageUrlString: String) {
+    func configure(image: UIImage) {
```

Comment:
```suggestion
    func configure(image: UIImage) {
```
UI нельзя передавать